### DISCUSSION TITLE
juicefs + minio storage space is not released

### DISCUSSION THREAD

**Discussion by Johnnyliu007** [juicedata/juicefs#2362] [General]
minio: /data/minio/run/minio server --config-dir /etc/minio /data/minio/data
format: juicefs format     --storage minio     --bucket http://127.0.0.1:9000/test-b-002     --access-key minio     --secret-key minio123     "mysql://mysqluser:mysqlpwd@(ip:port)/juicefs" pics
mount: juicefs mount -d "mysql://mysqluser:mysqlpwd@(ip:port)/juicefs" ~/jfs
create block file: dd if=/dev/zero of=test bs=1024M count=2
remove block file: rm test
the storage space is not released, du -sh . show 2.1G, the bucket keep that blk;
why?

**Comment by zhijian-pro** [comment]
The trash needs to be close.
https://juicefs.com/docs/zh/cloud/trash
